### ISSUE TITLE
No output 

### ISSUE THREAD

**Issue by CarbonMan** [Xyene/sphere2cube#3]
I am running sphere2cube under 64 bit windows. Blender is installed and in the path. I run the command
sphere2cube opt.jpg -fPNG
and it processes for about 5 seconds and returns but there is no output files and no messages. The input file is 18MB.

Can you suggest anything to try?

**Comment by Xyene** [comment]
Thanks for taking the time to report this, I've checked and it appears to have stopped working on my end as well. Unfortunately I won't have time in the next week to look further into it, but it could be that the way Blender initializes scripts has changed in a recent version &mdash; perhaps a slightly older version of Blender could work?

If you do figure it out, please let me know.

**Comment by guojia-git** [comment]
I faced a similar issue using the apt-get installed blender. Was able to fix it using [blender 2.77](http://download.blender.org/release/Blender2.77/) released in 2016.

**Comment by Wayde2014** [comment]
I also encountered this problem, running `sphere2cube source.hdr -fPNG` under windows without any output and no error. Found that the problem is not blender version, but the source file path problem, must enter the absolute path in windows, otherwise Blender will not find the file.
Blender has output error message, but `sphere2cube` does not get and output. I fixed and optimized these issues and passed the test. I have submitted a merge request.
